packages/jazz-tools/src/implementation/ContextManager.ts

```diff
@@ -43,10 +43,6 @@ export class JazzContextManager<
   protected context: PlatformSpecificContext<Acc> | undefined;
   protected props: P | undefined;
   protected authSecretStorage = new AuthSecretStorage();
-  protected authSecretStorageWithNotify = Object.assign(
-    Object.create(this.authSecretStorage),
-    { notify: true },
-  );
```

Comment:
This was the source of the bug.

Wanted to have a different notify the state when the AuthSecretStorage is used outside but did create also two different isAuthenticated states.